LocalWiki.php

```diff
@@ -309,7 +309,23 @@ function onBeforePageDisplay( OutputPage $out ) {
 		'url' => 'https://commons.miraheze.org/w/api.php'
 	];
 }
-
+// Image repository for polandballwikisongcontestwiki
+if ( $wgDBname === 'polandballwikisongcontestwiki' ) {
+	$wgForeignFileRepos[] = [
+		'class' => 'ForeignDBViaLBRepo',
+		'name' => 'shared-polcomwiki',
+		'directory' => '/mnt/mediawiki-static/polcomwiki',
+		'url' => 'https://static.miraheze.org/polcomwiki',
+		'hashLevels' => $wgHashedSharedUploadDirectory ? 2 : 0,
+		'fetchDescription' => true,
+		'descriptionCacheExpiry' => 86400 * 7,
+		'thumbScriptUrl' => false,
+		'transformVia404' => !$wgGenerateThumbnailOnParse,
+		'hasSharedCache' => 'polcomwiki',
```

Comment:
This is boolean, should be false

Reply:
This has been fixed, please review when you have the time, thanks!